src/main/webapp/WEB-INF/view/searchResults.jsp

```diff
@@ -1,5 +1,6 @@
 <%@page pageEncoding="UTF-8" contentType="text/html; charset=UTF-8"%>
 <%-- query matched some document --%>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.4/clipboard.min.js"></script>
```

Comment:
Fess cannot use files in CDN.
There are js files in src/main/webapp/js.

Reply:
I did not know that the use of a CDN was not allowed.
Thanks for letting me know.
I have corrected it.